setup.py

```diff
@@ -128,6 +128,7 @@
         "holidays>=0.10.3",
         "pyaml>=20.4.0",
         "redis>=3.5.3",
+        "redis-server>=6.0.6"
```

Comment:
To handle the issue same as https://stackoverflow.com/questions/36088409/error-111-connecting-to-localhost6379-connection-refused-django-heroku

Use `sudo apt-get install redis-server` to solve the issue met before. I'm not sure whether it is OK to add the dependency here.

@chaosddp